app/controllers/configuration_profile_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ConfigurationProfileController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::BreadcrumbsMixin
+
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def self.table_name
+    @table_name ||= "configuration_profile"
```

Comment:
We should be able to replace these with

```
def self.model
  ConfigurationProfile
end
```

(`table_name` defaults to `model.name.underscore`)

(technically, even the model method is not needed as the default implementation will guess from the controller name, but... :))